exotations/task_maps/task_map/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ add_definitions(-DEXOTICA_DEBUG_MODE)
 
```

Comment:
I just checked the content and forgot that dmesh_ros was also in that package, makes absolute sense.